sdks/python/apache_beam/runners/portability/universal_local_runner_main.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import logging
+import sys
+import time
+
+from apache_beam.runners.portability import universal_local_runner
+
+
+def run(argv):
+  if argv[0] == __file__:
+    argv = argv[1:]
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-p', '--port',
```

Comment:
<!--thread_id:cc_139272713_t; commit:433a15d3542753b40699aa7185a37dd90f19c48e; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> The "default" is to throw an error due to unprovided arguments. (Removed the default for command_line.) It's standard convention for services to take a single -p/--port argument.

<!--section:body-->
Thanks for clarification